subsys/net/lib/openthread/platform/flash.c

```diff
@@ -62,17 +62,26 @@ otError utilsFlashErasePage(u32_t aAddress)
 {
 	struct flash_pages_info info;
 	u32_t address;
+	otError err;
```

Comment:
you could define it as otError err = OT_ERROR_NONE;

Reply:
Right, sounds fine.